analytics/tests/test_activity_views.py

```diff
@@ -125,13 +161,31 @@ def test_activity(self, unused_mock: mock.Mock) -> None:
             is_renewal=True,
             plan=plan,
         )
-        RemoteZulipServer.objects.create(
+        server = RemoteZulipServer.objects.create(
             uuid=str(uuid.uuid4()),
             api_key="magic_secret_api_key",
             hostname="demo.example.com",
             contact_email="email@example.com",
         )
-        with self.assert_database_query_count(15):
+        extra_data = {
+            RemoteRealmAuditLog.ROLE_COUNT: {
+                RemoteRealmAuditLog.ROLE_COUNT_HUMANS: {
+                    UserProfile.ROLE_REALM_ADMINISTRATOR: 1,
+                    UserProfile.ROLE_REALM_OWNER: 1,
+                    UserProfile.ROLE_MODERATOR: 1,
+                    UserProfile.ROLE_MEMBER: 1,
+                    UserProfile.ROLE_GUEST: 1,
+                }
+            }
+        }
+        RemoteRealmAuditLog.objects.create(
+            server=server,
+            realm_id=10,
+            event_type=RemoteRealmAuditLog.USER_CREATED,
+            event_time=timezone_now() - timedelta(days=1),
+            extra_data=extra_data,
+        )
```

Comment:
This just covers broad strokes, so it would be good to write a more complete/correct test data set up here.